access reviews to v1 - using doc stub generator (#12207)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/removeaccessapplyaction.md

---
title: "removeAccessApplyAction resource type"

> description: "Type of accessReviewApplyAction that indicates removing access to a resource of those denied in an access review of that resource."

Reviewer: ```suggestion
description: "Removes access to a resource for those denied in an access review."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# removeAccessApplyAction resource type

Namespace: microsoft.graph

> removeAccessApplyAction is a kind of [accessReviewApplyAction](../resources/accessreviewapplyaction.md) that indicates removing access to a resource of those denied in an access review of that resource. This is the default option chosen for [accessReviewApplyAction](../resources/accessreviewapplyaction.md) if an option is not passed in.

Reviewer: ```suggestion
Removes access to a resource for those denied in an access review of that resource. This is the default option chosen for [accessReviewApplyAction](../resources/accessreviewapplyaction.md) if an option is not passed in.
```
